honeybee_radiance_command/options/ra_gif.py

```diff
@@ -0,0 +1,105 @@
+"""ra_gif parameters."""
+from .optionbase import OptionCollection, BoolOption, NumericOption, IntegerOption
+
+
+class Ra_GIFOptions(OptionCollection):
+    """ra_gif command options.
+
+    Also see: https://floyd.lbl.gov/radiance/man_html/ra_gif.1.html
+    """
+    __slots__ = ('_b', '_d', '_c', '_g', '_e', '_n')
+
+    def __init__(self):
+        """ra_gif command options."""
+        OptionCollection.__init__(self)
+        self._b = BoolOption("b", "Change image color - default: False")
+        self._d = BoolOption("d", "Turn off dithering - default: False")
+        self._c = NumericOption("c", "Fewer colors", min_value=1, max_value=256)
+        self._g = NumericOption("g", "Gamma correction - default: 2.2")
+        self._e = IntegerOption("e", "Exposure compensation")
+        self._n = NumericOption("n", "Sampling factor for large images", min_value=1,
+                                max_value=80)
+
+    @staticmethod
+    def _is_power_of_two(value):
+        """True if a given value is power of two."""
+        while (value % 2 == 0):
+            x = value / 2
+        return x == 1
```

Comment:
Can't we use math module?

```python
import math

math.log2(n).is_integer()
```

From here: https://stackoverflow.com/a/57025941/4394669